sdk/eventhub/azure-eventhubs/azure/eventhub/sender.py

```diff
@@ -60,6 +61,7 @@ def __init__(self, client, target, partition=None, send_timeout=60, keep_alive=N
         self.error = None
         self.keep_alive = keep_alive
         self.auto_reconnect = auto_reconnect
+        # max_retries = client.config.retry_policy.max_retries
         self.retry_policy = errors.ErrorPolicy(max_retries=3, on_error=_error_handler)
```

Comment:
I think this retry policy should now be built from the configuration?

Reply:
changed to max_retries=self.client.config.max_retries